test/rest.test.js

```diff
@@ -1359,6 +1359,112 @@ describe('strong-remoting-rest', function() {
           });
       });
 
+      it('should allow customized xml root element', function(done) {
+        var method = givenSharedStaticMethod(
+          function bar(cb) {
+            cb(null, {a: 1, b: 2});
+          },
+          {
+            returns: { arg: 'data', type: 'object', root: true,
+            xml: { wrapperElement: 'foo'} },
```

Comment:
please reindent these two lines to make it clear that `xml` is nested under `returns`:

```js
returns: {
  arg: ... root: true,
  xml: {...}
},
http: {...}
```